overrides.nix

```diff
@@ -615,6 +615,16 @@ self: super:
     }
   );
 
+  jq = super.jq.overridePythonAttrs (attrs: {
+    buildInputs = [ pkgs.jq ];
+    patches = [
+      (pkgs.fetchpatch {
+        url = "https://raw.githubusercontent.com/NixOS/nixpkgs/088da8735f6620b60d724aa7db742607ea216087/pkgs/development/python-modules/jq/jq-py-setup.patch";
```

Comment:
This won't work for us as `poetry2nix` overrides needs to be polymorphic (meaning that a single override supports multiple versions).

Reply:
Is the solution to only patch for versions that require the patch?

Reply:
Just had a more proper look at this.

While I'm not a fan of having version-specific patches that will eventually fail to apply, the build will fail regardless since they are doing insane things in the build.

I think the patch application failing is the lesser of two evils here.

Reply:
Ideally nixpkgs would upstream the patch. Did you consider this @rmcgibbo?

Reply:
No, I don't believe I tried to upstream it.